### PR TITLE
Add preferredPathSeparator setting

### PR DESCRIPTION
**Pull request by sean-mcmanus**

### CODE REVIEW

**Comment by sean-mcmanus** [comment]
The change seems to work, without any typescript changes.

**Review by bobbrow — APPROVED** [review]
We usually send the settings the language server needs down in the createLanguageClient method in client.ts.  Perhaps this is unnecessary and can be removed in a future update since we get a didChangeConfiguration event shortly after that and we could delay initialization until then.  Until we change this though, it might be best to follow the current pattern.  I don't feel strongly about it though.

**Comment by sean-mcmanus** [comment]
Sure, I followed the current pattern -- also updated the package.json version and fwlinks . There npm added some update package and the package-lock.json changed...not sure exactly why.

File: Extension/package.json [hunk 1]
```diff
@@ -1295,6 +1295,7 @@
     "minimatch": "~3.0.4",
     "mkdirp": "~0.5.1",
     "tmp": "~0.0.33",
+    "update": "^0.7.4",
```
Comment (WardenGnaw) [inline Extension/package.json]: What is this package used for?

This is the one adding a lot to the package-lock.json.
It goes from 182652 bytes to 414669 bytes.

**Comment by sean-mcmanus** [comment]
@WardenGnaw Do you understand the package.json update package change and the changes to package-lock.json?

**Review by WardenGnaw — CHANGES_REQUESTED** [review]
If we don't use the update package or have an updatefile.js, please remove the update package.

**Review by WardenGnaw — APPROVED** [review]

**Comment by sean-mcmanus** [comment]
The fwlinks all currently point to the same 0.16.1 binaries, but the targets will be updated when they're ready.

**Pull request merged**